docs/proposals/dehydration.md

```diff
@@ -0,0 +1,147 @@
+# Cluster Dehydration
+
+**Author**: Marvin Beckers (@embik)
+
+**Status**: Draft proposal.
+
+## Goals
+
+Dehydration is a functionality for KKP user clusters that will allow to temporarily "pause" a cluster to free up cloud resources used by the cluster. The feature will allow to optimize resource consumption and subsequent spending on cloud resources. State (e.g. storage) should not be affected by this functionality.
```

Comment:
Either use a different word than "pause" or make it clear that this has nothing to do with the existing pause mechanism, which simply disables reconciling but leaves the cluster up and running.

Reply:
"Pause" has been replaced with "Suspend".